src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsSplit.js

```diff
@@ -0,0 +1,258 @@
+        }
+
+        sections.push({
+            title: 'RECENTS',
+            data: this.state.recentReports,
+            shouldShow: this.state.recentReports.length > 0,
+            indexOffset: sections.reduce((prev, {data}) => prev + data.length, 0),
+        });
+
+        sections.push({
+            title: 'CONTACTS',
+            data: this.state.personalDetails,
+            shouldShow: this.state.personalDetails.length > 0,
+            indexOffset: sections.reduce((prev, {data}) => prev + data.length, 0),
+        });
+
+        if (this.state.userToInvite) {
+            sections.push(({
+                undefined,
+                data: [this.state.userToInvite],
+                shouldShow: true,
+                indexOffset: 0,
+            }));
+        }
+
+        return sections;
+    }
+
+    /**
+     * Once a single or more users are selected, adds the selected user emails with the request
+     */
+    addParticipants() {
+        const userEmails = this.state.selectedOptions.map(selectedOption => selectedOption.login);
+        this.props.onStepComplete(userEmails);
+        if (this.props.isSmallScreenWidth) {
+            hideSidebar();
+        }
+    }
+
+    /**
+     * Removes a selected option from list if already selected. If not already selected add this option to the list.
+     * @param {Object} option
+     */
+    toggleOption(option) {
+        this.setState((prevState) => {
+            const isOptionInList = _.some(prevState.selectedOptions, selectedOption => (
+                selectedOption.login === option.login
+            ));
+
+
+            let newSelectedOptions;
+
+            if (isOptionInList) {
+                newSelectedOptions = _.reject(prevState.selectedOptions, selectedOption => (
+                    selectedOption.login === option.login
+                ));
+            } else {
+                newSelectedOptions = [...prevState.selectedOptions, option];
+            }
+
+
+            const {
+                recentReports,
+                personalDetails,
+                userToInvite,
+            } = getNewGroupOptions(
+                this.props.reports,
+                this.props.personalDetails,
+                isOptionInList ? prevState.searchValue : '',
+                newSelectedOptions,
+            );
+
+            return {
+                selectedOptions: newSelectedOptions,
+                recentReports,
+                personalDetails,
+                userToInvite,
+                searchValue: isOptionInList ? prevState.searchValue : '',
+            };
+        });
+    }
+
+
+    render() {
+        const maxParticipantsReached = this.state.selectedOptions.length === CONST.REPORT.MAXIMUM_PARTICIPANTS;
+        const sections = this.getSections(maxParticipantsReached);
+
+        return (
+            <View style={[styles.flex1, styles.w100]}>
+                <Text style={[styles.headerText, styles.pt3, styles.ph5]}>
+                    TO
```

Comment:
Let's just make this `To` for now and use the style `styles.formLabel`

Reply:
Thank you! will implement the change